docs/manuals/characterization/stark_experiment.rst

```diff
@@ -145,6 +145,14 @@ by a variant of the Hahn-echo pulse sequence [5]_.
 
     %matplotlib inline
 
+    import warnings
```

Comment:
Following the policy we should deprecate everything in public. However, since experiment internally instantiates analysis, user will see the warning twice. I think this is a bit annoying. I think it's better to add some deprecation message to the tutorial since the tutorial will be also removed with experiment classes.

Reply:
Since there were not too many cases, I tried just catching the analysis warnings in the deprecated experiments so the experiment only emits one warning but the analysis class will still emit a warning otherwise (and get the warning that shows up in the docs).